### PR TITLE
Fix tests and add CI

### PR DESCRIPTION
**Pull request by thazhemadam**
* Add the packages required to use `FlightSims.jl` to its `Project.toml`
* Running `] test` would throw the following errors -
  1.  `figures` directory not being in the `test` directory.
  2. `integrated_environments.jl` includes a non-existent file (`backstepping_position_controller_static_allocator_multicopter_env.jl`)  instead of `backstepping_position_controller_static_allocator_multicopter.jl`.

Both these issues have been fixed now.

* Since there's no CI set up, there's no real way to know if any changes made to the package would break something else or not. Hence, add CI.

Note - `The Xvfb - install and run` step in the CI workflow is important. If this step isn't there, the following error occurs - 
```error
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: linuxfb, minimal, offscreen, vnc, xcb.

Aborted (core dumped)
connect: Connection refused
GKS: can't connect to GKS socket application


signal (11): Segmentation fault
...
```

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -4,10 +4,14 @@ authors = ["JinraeKim <kjl950403@gmail.com> and contributors"]
 version = "0.10.0"
 
 [deps]
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```
Comment (JinraeKim) [inline Project.toml]: I think it's only for test; do we have to add these dependencies?

**Review by JinraeKim — CHANGES_REQUESTED** [review]
@thazhemadam Thanks for your effort 👍 

Honestly, I'm not familiar with continuous integration (CI).
~~Could you explain it briefly or refer to any sources?~~
EDIT: I roughly understood its concept. Very roughly, it literally helps us to integrate code.
So, the only thing we have to do is simply adding the `CI.yml` in this repo? Could you explain details, e.g., which CI software is used?

Please refer to reviews for other comments.

**Comment by thazhemadam** [comment]
@JinraeKim simply adding the CI.yml file works in this case as the CI workflow is correctly configured. It is run using [Github Actions](https://github.com/features/actions). The way it is currently set up, the CI runs every time a PR is made to `master` (this can be regulated by preventing first-time contributors' PRs to trigger CI runs) or a commit is pushed directly to master.

File: Project.toml (hunk 1 above)
Comment (thazhemadam) [reply Project.toml]: Ah, okay. 6894d5a considers these dependencies as test-specific dependencies.

**Comment by JinraeKim** [comment]
@thazhemadam Ah, I understood how I should deal with test-specific packages now (so embarrassed due to the fact that I haven't known it by now).

Thanks for your kind explanation and helpful commits!

**Review by JinraeKim — APPROVED** [review]
Thanks, @thazhemadam !

**Comment by JinraeKim** [comment]
@thazhemadam Hi, again!

I'd like to know how to pre-test the CI workflow before merging it into `origin/master`.
That is, is it possible to test the CI in my custom branch?

**Comment by thazhemadam** [comment]
Hi, @JinraeKim! Opening a pull request from your branch to `master` will trigger the CI run. Once the PR is opened, the CI will also run for every commit you make to your branch subsequently.
(see [here](https://github.com/JinraeKim/FlightSims.jl/blob/master/.github/workflows/CI.yml#L3-L5))

**Comment by JinraeKim** [comment]
Thank you, @thazhemadam.
But I think that there was a miscommunication.
I mean, before adding the workflow to the branch annotated (in this case, `master`), can I check whether the CI will work properly?
For example, if I add the file to a branch `foo` and annotate it as to be tested for all PRs, it will work not even for `foo != master`?

**Comment by JinraeKim** [comment]
@thazhemadam
I found that PR of branch `foo` to `master` with `CI.yml` automatically runs the github workflow.
Thanks :)

**Pull request merged**